bin/node/runtime/src/lib.rs

```diff
@@ -738,6 +779,7 @@ parameter_types! {
 			<Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(1) -
 			<Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(0)
 		)) / 5) as u32;
+	pub MaxCodeSize: u32 = 128 * 1024;
```

Comment:
“All entry points pre-charge the weight for accessing a maximum size code and correct this weight after execution.”
[link to the PR description](https://github.com/paritytech/substrate/pull/8086)